### PR TITLE
Go to section feature

### PR DESCRIPTION
**Pull request by PabloFLPs**
Adding "go to" feature to items on the NavigationBar and menu items (for mobile viewing), in addition, I also added a smoothness property to the scrolling.

### CODE REVIEW

File: src/components/NavigationBar.tsx [hunk 1]
```diff
@@ -41,30 +41,25 @@ export default function NavigationBar() {
                   {/* Current: "border-indigo-500 text-gray-900", Default: "border-transparent text-gray-500 hover:border-gray-300 hover:text-gray-700" */}
                   <a
                     href="#"
+                    onClick={() => "window.scrollTo(0,0)"}
```
Comment (Erickinhou) [inline src/components/NavigationBar.tsx]: This is necessary in this case?? The `href= '#'` do it I think.

File: src/components/NavigationBar.tsx [hunk 2]
```diff
@@ -80,37 +75,31 @@ export default function NavigationBar() {
               <Disclosure.Button
                 as="a"
                 href="#"
+                onClick={() => "window.scrollTo(0,0)"}
```
Comment (Erickinhou) [inline src/components/NavigationBar.tsx]: same as above

**Review by Erickinhou — APPROVED** [review]
Everything is good! Just one question.
I don't know why but the selection on the navbar is looking weird, when I click on it the underline selection doesn't work properly.

**Review by Erickinhou — COMMENTED** [review]
Peek 2022-02-25 01-28

File: src/components/NavigationBar.tsx (hunk 1 above)
Comment (PabloFLPs) [reply src/components/NavigationBar.tsx]: Oh, I didn't realize that! Removed :)

**Pull request merged**